packages/babel-core/src/transformation/plugin.js

```diff
@@ -1,15 +1,12 @@
 import OptionManager from "./file/options/option-manager";
 import * as messages from "babel-messages";
-import Store from "../store";
 import traverse from "babel-traverse";
 import clone from "lodash/clone";
 
 const GLOBAL_VISITOR_PROPS = ["enter", "exit"];
 
-export default class Plugin extends Store {
```

Comment:
`this` in that example is an instance of `PluginPass`, not `Plugin`. The only place `Plugin` was ever even exposed was `this.plugin` which I'm removing in this PR too, so removing the subclassing won't hurt anything that isn't already broken by that.